doc/getting-started.md

```diff
@@ -82,24 +93,65 @@ most simple capabilities is to add a delay.
 
 ```
 # First, create a rule file
-cat <<EOF > /tmp/hello-5s-rule.yaml
```

Comment:
Also my bad for not noticing this earlier. Showcasing a delay as the first Istio capability seems odd. I would suggest showcasing something with content based routing or %-based traffic split